app/posts/manage-teacher-training-applications/2021-04-08-gathering-school-direct-contact-details-from-higher-education-institutions.md

```diff
@@ -0,0 +1,122 @@
+
+Since the previous iteration, we’d discovered that HEI users have to sign a data sharing agreement (DSA) in order to get access to the ‘sandbox’ version of Manage. They access this so that they can experiment before signing up to use the service with real data.
+
+Since each HEI should have already signed a DSA, we decided not to ask them to do it as part of our process.
+
+## Research findings
+
+Users understood what they were being asked to do in the email.
+
+Some users said they would not want to complete the spreadsheet immediately. They’d want to make sure they were ready to use Manage and also let SDs know what was happening.
+
+When we asked users to complete the spreadsheet, we found that in general they: 
+
+- expected to invite all their SDs at the same time
+- did not recognise the provider code we used – some expected the unique reference number (URN) instead
+- wanted to see all their SDs listed, including the ones which were already set up in Manage 
+- would add any missing information, such as SDs which should have been on the list 
+- were unsure about what to do if something was incorrect
+- would choose to complete and return the CSV file rather than use the Google Sheets version
+
+## Future considerations
+
+We are confident that users would be able to complete the spreadsheet we tested. 
+
+Before using it, we’d need to give more thought to: 
+
+- how users could tell us about inaccuracies
+- what would happen if a user changed information in the spreadsheet or added extra SDs
+- the reference number we’d use in the spreadsheet
+- whether to offer a Google Sheets version
+- how to be clearer in the email about which courses would become available and when
+
+After the research we decided to focus on the next stage of the process. We’ll need a more streamlined way to set up users once we've gathered contact details from HEIs.
+
+{% from "screenshots/macro.njk" import appScreenshots with context %}
+{{ appScreenshots({
+  items: [{
+    text: "Spreadsheet for gathering SD contact details",
+    img: {
+      src: "sd-contact-details-spreadsheet.png"
+    }
+  }]
+}) }}
+
+## Emails
+
+### Asking users to give SD contact details
+
+<!-- markdownlint-disable MD025 MD001 -->
+{{ appEmail({
+ subject: "Give contact details for your School Direct partners",
+ content: "
+
+Dear Andrew
+
+Some teacher training courses accredited by Bishop Grosseteste University are not available through GOV.​UK.
+
+This is because each School Direct partner needs to set up an online account before their courses become available.
+
+# What you need to do
+
+We’ve created a spreadsheet listing the partners whose courses you accredit.
+
+There’s space for you to add the name and email address of your main contact at each partner which does not have an online account.
+
+You can either:
+
+- [download the spreadsheet as a CSV file](), fill it in and return it to providercontacts@digital.education.gov.uk
```

Comment:
Can you add a `#` to the URL so as not to leave blank, albeit changing the linting rules might be fine.